docs/Installing.asciidoc

```diff
@@ -311,6 +310,23 @@ NFS clients are where openQA workers are running. Run following command:
 mount -t nfs openQA-webUI-host:/var/lib/openqa/share /var/lib/openqa/share
 --------------------------------------------------------------------------------
 
+Needle Caching  
+~~~~~~~~~~~~~~~
+
+If your network is slow or you experience long time to load needles you
+might want to consider needle caching. To use needle caching a directory
```

Comment:
then make it show them correctly :-) Are you sure that this code differs from what you have locally?